constraint/pkg/client/backend.go

```diff
@@ -44,7 +44,7 @@ func NewBackend(opts ...BackendOpt) (*Backend, error) {
 }
 
 // NewClient creates a new client for the supplied backend.
-func (b *Backend) NewClient(opts ...Opt) (*Client, error) {
+func (b *Backend) NewClient(ctx context.Context, opts ...Opt) (*Client, error) {
```

Comment:
Is NewClient a place we want to inject context?

Reply:
Like above, we don't actually use the context for anything so it just pollutes call sites. I've removed it.